Automattic/woocommerce-payments
# Show empty dispute details component on Transaction Details page

Fixes https://github.com/Automattic/woocommerce-payments/issues/6886.

#### Changes proposed in this Pull Request

- Add a feature flag for dispute details UI on Transaction Details page.
- When feature flag is enabled, show an empty dispute details UI.

#### Testing instructions

- Make a local modification of `DisputeDetails`'s return value in `client/payment-details/dispute-details/index.tsx` to show something. For example:

```diff
- 	return <></>;
+ 	return <span>abc</span>;
```

- Enable feature flag by running `update_option( '_wcpay_feature_dispute_on_transaction_page', '1' );` in [WP Console](https://wordpress.org/plugins/wp-console/) or by modifying your database table `wp_options` directly.
- Load a Transaction Details page (`/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Ftransactions%2Fdetails&id=pi_abc&transaction_id=txn_abc&transaction_type=charge`).
- Make sure something shows up at the bottom of the payment summary card.

-------------------


**Post merge**
